core/evancore.vue.libs/src/components/forms/control/control.ts

```diff
@@ -34,7 +34,7 @@ export default class ControlComponent extends mixins(EvanComponent) {
   /**
    * The value for the input field.
    */
-  @Prop() value: Array<any> | string;
+  @Prop() value: any;
```

Comment:
Why was this necessary?

Reply:
By adding the json form control, specially any value can be passed through the control it self and the derived component must decide, which values are allowed.